README.md

```diff
@@ -19,29 +18,23 @@ On subsequent visits to the community business, the user can quickly sign in wit
 The local business owners will then be able to retrieve information about the popularity of their different offers, and whether this changes according to demographic groups. This can help them tailor their services to the needs and preferences of their local community.
 
 ![Datapower walkthrough](https://user-images.githubusercontent.com/23265724/31492017-33806fd0-af41-11e7-9af8-a38a424dc906.gif)
-(*Oops, spot the Kilgore/Kilgrave mistake...*)
-
-## How Datapower works
 
-The website is built in [React](https://reactjs.org/) with [Create React App](https://github.com/facebookincubator/create-react-app), using an [Express](https://expressjs.com/) server and a [PostgreSQL](https://www.postgresql.org/) database.
-
-The QR code is generated using the [qrcode](https://github.com/soldair/node-qrcode) node module, and the scanning is then run with the [Instascan](https://github.com/schmich/instascan) HTML5 QR webcam scanner.
+(*Oops, spot the Kilgore/Kilgrave mistake...*)
 
 ## Documentation
 Please see the [docs](./docs) directory for project documentation.
 
-## Useful links
-
-1. [Create React App](https://github.com/facebookincubator/create-react-app)
-2. [Advice on using React with an Express server](https://daveceddia.com/create-react-app-express-backend/)
-3. [Create React App with Express in Production](https://daveceddia.com/create-react-app-express-production/)
-4. [React Router](https://reacttraining.com/react-router/)
-5. [Codeacademy Course on React](https://www.codecademy.com/learn/react-101)
+## Contributors
+Brought to you by:
+* [@azayneeva](https://github.com/azayneeva)
+* [@RogeredBacon](https://github.com/RogeredBacon)
+* [@rachaelcodes](https://github.com/rachaelcodes)
+* [@ZooeyMiller](https://github.com/ZooeyMiller)
+* [@finnhodgkin](https://github.com/finnhodgkin)
+* [@astroash](https://github.com/astroash)
+* [@eliascodes](https://github.com/eliascodes)
 
 ## Thanks
-
-* Our Mothers and Fathers for bringing us into this world, and supporting us through our mad *I'm a developer now!* epoch.
```

Comment:
It's mothers day in Palestine today. Feels like a kinds cutting time to rip this out of our readme.